### ISSUE TITLE
Tutorial 2. Object detection -  Installation Prerequisistes dependencies

### ISSUE THREAD

**Issue by ravisidia** [nnstreamer/nnstreamer#4259]
Hello,
I'm trying to run the tutorials, and in the installation of nnstreamer-tensorflow it ask for some dependencies that is not mentioned on the installation page. 

>>sudo apt install wget nnstreamer nnstreamer-example nnstreamer-tensorflow2-lite
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
wget is already the newest version (1.21.2-2ubuntu1).
wget set to manually installed.
nnstreamer is already the newest version (2.4.0.0-0~202309150847~ubuntu22.04.1).
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

**The following packages have unmet dependencies:
 nnstreamer-tensorflow : Depends: tensorflow but it is not installable or
                                  tensorflow-dev but it is not installable**
E: Unable to correct problems, you have held broken packages.^

Anyone can help me with this problem?

**Comment by gichan-jang** [comment]
./get-model.sh object-detection-tflite

**Comment by ravisidia** [comment]
> Thanks for reporting this issue. I will fix `nnstreamer-example` package issue what you mentioned. To do tutorial 2, there is workaround. Down load this [script](https://github.com/nnstreamer/nnstreamer-example/blob/main/bash_script/example_models/get-model.sh) and get the model manually.
> 
> ```
> $ ./get-model.sh object-detection-tflite
> ```
> 
> And then run the pipeline. Please care about the model and label path.

Hello @gichan-jang, thanks for the help.
I forgot to mention that I'm using Ubuntu 22.04 because I could no install in 20.04. I made what you said end other error occured

> gst-launch-1.0 \
    v4l2src ! videoconvert ! videoscale ! video/x-raw,width=640,height=480,format=RGB,framerate=30/1 ! tee name=t \
    t. ! queue leaky=2 max-size-buffers=2 ! videoscale ! video/x-raw,width=300,height=300,format=RGB ! tensor_converter ! \
            tensor_transform mode=arithmetic option=typecast:float32,add:-127.5,div:127.5 ! \
            tensor_filter framework=tensorflow-lite model=tflite_model/ssd_mobilenet_v2_coco.tflite ! \
            tensor_decoder mode=bounding_boxes option1=mobilenet-ssd option2=tflite_model/coco_labels_list.txt option3=tflite_model/box_priors.txt option4=640:480 option5=300:300 ! \
            compositor name=mix sink_0::zorder=2 sink_1::zorder=1 ! videoconvert ! autovideosink \
    t. ! queue leaky=2 max-size-buffers=10 ! mix.
**WARNING: erroneous pipeline: no element "tensor_converter"**

**Comment by gichan-jang** [comment]
You should install nnstreamer.
```
$ sudo apt install nnstreamer nnstreamer-tensorflow2-lite tensorflow2-lite-dev 
```
To check if nnstremaer is installed properly, use `gst-inspect-1.0` tool.
```
$ gst-inspect-1.0 tensor_converter
Factory Details:
  Rank                     none (0)
  Long-name                TensorConverter
  Klass                    Converter/Tensor
  Description              Converts an audio, video, text, or arbitrary stream to a tensor stream of C-Array for neural network framework filters
  Author                   MyungJoo Ham <myungjoo.ham@samsung.com>
 .....
``

**Comment by gichan-jang** [comment]
nnstreamer-example debian package is fixed.
